homeassistant/components/reolink/select.py

```diff
@@ -70,6 +70,7 @@ class ReolinkSelectEntityDescription(
         key="ptz_preset",
         translation_key="ptz_preset",
         icon="mdi:pan",
+        entity_category=EntityCategory.CONFIG,
```

Comment:
This will actually move the camera to the preset position, therefore this is a control, not a config (I think)